### DISCUSSION TITLE
How about i18n support?

### DISCUSSION THREAD

**Discussion by AkiaCode** [FunkinCrew/Funkin#1872] [Ideas]
I wish me would provide language support for other countries

**Comment by AbnormalPoof** [comment]
what?

**Reply by isakube** [reply]
translations, miku pfp

**Comment by steviegt6** [comment]
As it stands, i18n and L10n may be tough, due to the custom-curated font and individual assets containing what _should_ be translatable text. Would be nice to have some sort of backup language, but the Funkin' font is custom-rendered and doesn't work identically to a normal font face.
